documentation/docs/developers/strategies/twap.md

```diff
@@ -0,0 +1,38 @@
+# TWAP
+
+## Overview
+
+TWAP strategy expands upon Simple Trade strategy. By the end of this part, you should be able to : 
+
+* Divide up an order into *n* individual orders
+* Add time delay between each individual orders
```

Comment:
While these are new features in the TWAP strategy, these are not new things that a reader may learn from reading the strategy code and documentation.

The technically new things in the TWAP strategy are these:

 1. We're emitting multiple orders over time by adding state transitions to `c_process_market()`. Basically this serves as an entry point for users to learn how to create sophisticated behaviors over time in strategies by constructing state machines.
 2. The need to quantize order size in practical trading algorithms.

The big difference of TWAP strategy from previous toy strategies should be that it actually implements something practical (i.e. a common algorithmic execution method). So introducing it by saying it's derived from the simple trade strategy misses the point.

So the introduction section needs to be rewritten... you should start by saying how this is different from the previous toy examples to motivate the reader. Then, talk about the new things in the TWAP strategy - from the reader's perspective, not from a "writing a report" perspective.